policy/lib/tekton/task.rego

```diff
@@ -99,7 +99,7 @@ task_names(task) := names if {
 	names := {name} | params
 }
 
-# task name from a v0.1 and v0.2 attestation
+# task name from a v0.1, v0.2 and v1.0 attestation
```

Comment:
I don't know if `v0.1` is true. I don't think we ever dealt with a version of Chains that emitted that.